src/components/personkort/PersonkortHeader/PersonkortHeader.tsx

```diff
@@ -54,9 +73,15 @@ const PersonkortHeader = () => {
           alt="person"
         />
         <div>
-          <h3>{`${navbruker.navn} (${hentBrukersAlderFraFnr(
-            personident
-          )} år)`}</h3>
+          <PersonkortH3>
+            {`${navbruker.navn} (${hentBrukersAlderFraFnr(personident)} år)`}
```

Comment:
Forresten så er det sånn at fødselsnummer nødvendigvis ikke kommer til å ha rett informasjon om hverken alder eller kjønn fra 2032. Så i fremtiden så går det an å hente disse feltene fra PDL i stedet for å utlede fra fødselsnummer 😬 
Ref https://pdldocs-navno.msappproxy.net/ekstern/index.html#_ny_identifikator_fra_folkeregisteret_innen_1_januar_2032

Reply:
Åja, da får du lage litt mer PDL-kode da 😄

Reply:
Some day in the future, maybe 🔮 🌈